### ISSUE TITLE
Adding multiple Y Axis to Scatter Plot

### ISSUE THREAD

**Issue by kkaiser41** [ScottPlot/ScottPlot#2203]
---------------------------------

ScottPlot Version: 4.1.50

Operating System: Windows

Application Type: WinForms

I have a question about adding multiple Y axis to a scatter plot. I am trying to add multiple Y axis to the left edge. I have done this with a signal plot with no problems but I seem to be having a problem doing it with a scatter plot.

The first thing I noticed is that when I go to create a second axis with an index of 1 I get a run time error stating that positions 0 and 1 are occupied by default. After this I changed my code so that I would automatically start my second Y axis with an index of 1 and not actually create a new one until I had a third Y axis, subsequently I would then create new ones starting at index 2 and increasing with each axis. This resolved the run time error but resulted in some strange behavior.

Basically another axis is not created on the graph until I start creating new ones with an index 2 and then continue to increment with each additional axis. The problem then is each subsequent axis seems to get out of sync with the data and there always seems to be one axis with no data assigned.

Has anyone else tried this? I am guessing I am probably just doing it wrong but I don't know what is wrong as this seems to work fine for me with the signal plot.

Any help would be appreciated!!

**Comment by kkaiser41** [comment]
I figured out the issue. I obviously was not thinking clearly. Each line is being displayed it's just that the auto scale puts them on top of each other.

Sorry for missing this.
